doc/src/contribute/faq.md

```diff
@@ -63,15 +63,15 @@ Yes, the token name will be SUI.
 
 ### When is the Sui Devnet/Testnet/Mainnet launching?
 
-We have launched our [Sui Devnet](../build/devnet.md) in May 2022. A Testnet is coming in a few months. 
+We have launched our [Sui Devnet](../build/devnet.md) in May 2022. A Testnet is coming soon.
```

Comment:
We launched...
IBM Style Guide:
Grammar
Apply the following grammar guidelines when you write for an international audience:
• Write in active voice and the present tense as much as possible.